### PR TITLE
multi-language support for Accounts page and its subpages

### PR DESCRIPTION
**Pull request by ghost**
Description of changes:
image
image
image

image

Checklist:

- [x] Have you successfully deployed to an AWS account with your changes?
- [x] Have you successfully tested with your changes locally?

### CODE REVIEW

**Review by dengmingtong — COMMENTED** [review]
Please make sure all checks are successful
https://github.com/awslabs/service-workbench-on-aws-cn/actions/runs/2716184759

File: addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/accounts/AccountCard.js [hunk 1]
```diff
@@ -233,7 +240,7 @@ class AccountCard extends React.Component {
   renderBudgetButton() {
     return (
       <Button floated="right" color="blue" onClick={this.handleBudgetButton}>
-        Budget Detail
+        {i18next.t('budgetDetail')}
```
Comment (dengmingtong) [inline addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/accounts/AccountCard.js]: Suggest to move "budgetDetail" key into `main/solution/ui/public/locales/en/accounts.json` file

**Review by chenhaiyun — DISMISSED** [review]

File: addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/accounts/AccountCard.js (hunk 1 above)
Comment (ghost) [reply addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/accounts/AccountCard.js]: done in the latest update

**Comment by ghost** [comment]
The latest submission mainly translates some previously untranslated form content。
iShot_2022-07-25_17 47 04
iShot_2022-07-25_17 47 35
iShot_2022-07-25_17 54 24
iShot_2022-07-25_17 47 40
iShot_2022-07-25_17 48 00
iShot_2022-07-25_17 48 12
iShot_2022-07-25_17 55 01

**Review by dengmingtong — APPROVED** [review]

**Comment by luorobin-a2z** [comment]
1. 第一个界面的 Version Beta (Latest Version) 不能翻译么？
2. 第四个界面 “失效于12hours from  now" --> ”将于12个小时之后失效“

**Pull request merged**